src/plugins/intel_npu/src/al/include/npuw_private_properties.hpp

```diff
@@ -307,7 +307,9 @@ static constexpr ov::Property<bool> full{"NPUW_DUMP_FULL"};
  * Type: std::string.
  * Dump the specified subgraph(s) in OpenVINO IR form in the current directory.
  * Possible values: Comma-separated list of subgraph indices or "YES" for all
- * subgraphs, "NO" or just empty value to turn option off. E.g. "0,1" or "YES".
+ * subgraphs, "NO" or just empty value to turn option off. Keyword "last" can
+ * be used for dumping last subgraph without specifying it by specific indice.
```

Comment:
Nitpick: `indice` typo, but comment might be ignored

Reply:
that's fair to be `index`, why should this comment be ignored

Reply:
Just aligned with comment message above: 
> Possible values: Comma-separated list of subgraph indices or "YES" for all subgraphs

Should I change it too?

Reply:
Fixed